CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 The change log describes what is "Added", "Removed", "Changed" or "Fixed" between each release.
 
+# 1.23.1
```

Comment:
i usually put an ` - unreleased` on the upcoming version to be more explicit and not confuse people who look at the changelog on github and wonder why they did not get it.